contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -371,8 +375,8 @@ func (s *SchedulerServer) prepareExecutorInfo(hks hyperkube.Interface) (*mesos.E
 		log.Infof("Detected %d staticPods in Configuration.", numberStaticPods)
 
 		info.Resources = []*mesos.Resource{
-			mutil.NewScalarResource("cpus", float64(numberStaticPods)*podtask.DefaultContainerCpus),
-			mutil.NewScalarResource("mem", float64(numberStaticPods)*podtask.DefaultContainerMem),
+			mutil.NewScalarResource("cpus", float64(numberStaticPods)*float64(podtask.DefaultContainerCpus)),
```

Comment:
can we make the following adjustment as part of this PR:

1. info.Resources should allocate some cpu and memory to account for the fact that the kubelet-executor and kube-proxy are running on the slave; should happen regardless of whether there are static pods. something like DefaultExecutorMem and DefaultExecutorCPU.
1. if there are static pods, the resources from the above should be included in the calculation